aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -150,7 +150,42 @@ module aptos_framework::aptos_coin {
     }
 
     #[test_only]
-    use aptos_framework::aggregator_factory;
+    public fun mint_fa_for_test(amount: u64): FungibleAsset acquires MintCapStore {
+        ensure_initialized_with_fa_metadata_for_test();
+        coin::coin_to_fungible_asset(
+            coin::mint(
+                amount,
+                &borrow_global<MintCapStore>(@aptos_framework).mint_cap
```

Comment:
```suggestion
                &borrow_global<MintCapStore>(@aptos_fungible_asset).mint_cap
```

Per https://github.com/aptos-labs/aptos-core/pull/11224#discussion_r1516706137

Reply:
correct.

Reply:
The mint cap store is still at 0x1

Reply:
Got it, thank you for clarifying. Yes this looks good to me

Related: https://github.com/aptos-labs/aptos-core/pull/11224#discussion_r1518130755